src/infrastructure/repositories/figma-team-repository.integration.test.ts

```diff
@@ -28,7 +28,7 @@ describe('FigmaTeamRepository', () => {
 
 			const result = await figmaTeamRepository.findByWebhookId(webhookId);
 
-			expect(result).toEqual(figmaTeam);
+			expect(result).toMatchObject(figmaTeam);
```

Comment:
[See test failure](https://github.com/atlassian-labs/figma-for-jira/actions/runs/6672321052/job/18136002071).

result is a FigmaTeam. figmaTeam is a FigmaTeamSummary. result has some additional fields.